osf/models/preprint_provider.py

```diff
@@ -30,6 +30,7 @@ class PreprintProvider(ObjectIDMixin, BaseModel):
     social_twitter = models.CharField(null=True, blank=True, max_length=200)  # max length on prod: 8
     social_facebook = models.CharField(null=True, blank=True, max_length=200)  # max length on prod: 8
     social_instagram = models.CharField(null=True, blank=True, max_length=200)  # max length on prod: 8
+    blog = models.URLField(null=True, blank=True)
```

Comment:
Consider renaming to `blog_url`, to better follow naming conventions for `URLFields`. Also, should this have a `max_length`?